core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -548,6 +548,8 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     ConditionalPolicy("after adopting [policy]", UniqueTarget.Conditional),
     ConditionalNoPolicy("before adopting [policy]", UniqueTarget.Conditional),
 
+    ConditionalTimedUnique("for [amount] turns", UniqueTarget.Conditional),
```

Comment:
I think this unique should always return 'true' in the 'when' statement of `conditionalApplies`, as otherwise it will default to false and the unique will be ignored?

Reply:
Damn you're right